WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -8,13 +8,33 @@ class LoginPrologueViewController: LoginViewController {
 
     private var buttonViewController: NUXButtonViewController?
     var showCancel = false
+    var onLoginButtonTapped: (() -> Void)?
```

Comment:
This could probably use a simple description. 🙂

Reply:
Yup! added in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/373/commits/7f6b0f6c8729d77bb85f133541a70af019f146a6